common/src/main/resources/messages/messages.properties

```diff
@@ -122,6 +122,14 @@ cluster.gateway.change=Starting primary gateway change
 cluster.gateway.changed.successfully=Primary gateway successfully changed to {0}
 cluster.gateway.change.failed=Primary gateway change failed. Reason: {0}
 cluster.ambari.cluster.decommissioning.time=The decommission of the Data nodes will take approximately {0}.
+
+cluster.externaldatabase.deletion.started=External database termination started.
```

Comment:
external.database ?

Reply:
actually the current separation makes more sense logically